priv/templates/phx.gen.auth/schema.ex

```diff
@@ -68,6 +69,8 @@ defmodule <%= inspect schema.module %> do
       changeset<%= if hashing_library.name == :bcrypt do %>
       # If using Bcrypt, then further validate it is at most 72 bytes long
       |> validate_length(:password, max: 72, count: :bytes)<% end %>
+      # Hashing could be done with `Ecto.Changeset.prepare_changes/2`, but that
+      # would keep the database transaction open longer and hurt performance.
```

Comment:
I wondered why hashing was not done with `prepare_changes/2` and speculated this might be the reason. @mcrumm confirmed by finding https://github.com/aaronrenner/phx_gen_auth/issues/87